app/screens/image-gallery-viewer/gallery-image.tsx

```diff
@@ -43,13 +47,31 @@ const MAX_SCALE = 6
 const SWIPE_UP_THRESHOLD = 10
 const SWIPE_TO_CLOSE_THRESHOLD = 100
 
+import * as MediaLibrary from "expo-media-library"
+import type { Asset as Media } from "expo-media-library"
+
+const getUserAssets = async (): Promise<Media[] | null> => {
```

Comment:
@emadbaqeri  We have already defined this function on the asset-service.ts and you can use it

Reply:
As you can see I'm not using that anywhere in the code I should have remove it ;)
Thanks for mentioning by the way